.github/workflows/build.yml

```diff
@@ -135,6 +137,9 @@ jobs:
       - name: Install dependencies
         run: scoop install git python llvm cmake curl
 
+      - name: Install Mako
+        run: pip install mako
```

Comment:
Can you move this step to "Install dependencies"? Same for all other jobs and readme instructions.